orders/models.py

```diff
@@ -55,12 +70,21 @@ class Orders(models.Model):
     chosen_truck = models.CharField(max_length=100, choices=truck_types, null=True)
     pickup_date = models.DateField(null=True)
     pickup_time = models.TimeField(null=True)
+    delivery_date = models.DateField(null=True)
+    delivery_time = models.TimeField(null=True)
     need_packing = models.BooleanField()
     need_labor = models.BooleanField()
     order_state = models.CharField(
         max_length=100, choices=order_states, default="unassigned"
     )
     pricing = models.IntegerField(null=True)
 
+    @property
+    def distance(self):
+
```

Comment:
make a variable called distance=None 
by default 
in the if assign the value to it 
and then return it 
better than many Nones